mission_yaml/SEA45_M48.yml

```diff
@@ -373,11 +448,11 @@ profile_variables:
 
   instrument_ctd:
     comment:    unpumped RBR legato CTD
-    calibration_date:     "2021-03-01"
+    calibration_date:     "2020-07-23"
     calibration_report:  " "
```

Comment:
what is cal report?

Reply:
I think it's calibration report. I guess it would be a path to where the file is stored? As there's no way to embed a pdf in these. Another hangover from my hurried copy-pasting. Not useful imo. Let's just get rid of it

Reply:
I'm also not entirely sure why this stuff about the ctd needs to be repeated. We'll need to keep the block for now though, as the processing crashes without it!

Reply:
I think we don't need a path for a calibration report. Also why would it be only for one sensor? Let's skip this line

Reply:
I agree. Removed